### PR TITLE
✏️ Add pxswap project

### PR DESCRIPTION
**Pull request by alikonuk1**
##### Name (to be shown on DefiLlama):

pxswap

##### Twitter Link:

https://twitter.com/pxswap_xyz

##### Website Link:

https://pxswap.xyz

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):

logopx

##### Current TVL:

0

##### Chain:

Kava

##### Short Description (to be shown on DefiLlama):

NFT OTC Trading platform, the Future of NFT Finance

##### Category (full list at https://defillama.com/categories) *Please choose only one:

NFT 

##### methodology (what is being counted as tvl, how is tvl being calculated):

Overall colletral balance of the contract, nfts are not included

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/pxswap exports TVL: 
        
 
 
    ethereum                  0
    
    total                    0

File: projects/pxswap/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const sdk = require('@defillama/sdk')
+const { sumTokensExport } = require('../helper/unwrapLPs')
+
+const contract = "0x638B06F80FB28F109E65C9d5cC585aDf9A0c3f9f"
+const usdcToken = "0xfA9343C3897324496A05fC75abeD6bAC29f8A40f"
+
+module.exports = {
+    ethereum: { tvl: sumTokensExport({ owner: contract, tokens: [usdcToken]}) },
```
Comment (Define101) [inline projects/pxswap/index.js:8]: kava

**Comment by llamatester** [comment]
The adapter at projects/pxswap exports TVL: 
        
 
 
    kava                      0
    
    total                    0

**Comment by g1nt0ki** [comment]
I am very confused, why is the chain marked as kava, the website looks like aggregator on top ethereum. and I dont see any tvl, either in the contract on kava or on ethereum. Are people depositing NFTs/erc20s in your platform?

**Comment by alikonuk1** [comment]
If you navigate the website you can see that the part you see is just for navigating and not buying or selling nfts on Ethereum, Optimisim, Polygon and Goerli chains. Our real product is on Swap tab you see on the upper middle. In there you can put swap orders on Optimisim, Kava, Polygon and Goerli chains. So people are depositing both NFTs and ERC20 on our platform, we did not do the marketing yet so thats why we dont have some tvl yet @g1nt0ki

**Comment by g1nt0ki** [comment]
I am merging this PR, but till you have some tvl, UI will automatically hide it

**Pull request merged**